src/main/java/uk/gov/cshr/civilservant/domain/Answer.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.cshr.civilservant.domain;
+
+import java.util.Map;
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.*;
+import uk.gov.cshr.civilservant.mapping.AnswerMapConverter;
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Answer {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Size(min = 1, max = 10, message = "The answer should be less than 10 characters")
```

Comment:
Will this accept something length of 10 characters?

Reply:
Okay. this needs to be gone. good spot